omero/developers/Python.rst

```diff
@@ -38,6 +38,8 @@ which must be the same major version, i.e. |version|.x.
 All the code examples below can be found at
 :sourcedir:`examples/Training/python`.
 
+More complete workflow examples can be `found below  <#example-workflow-scripts>`_.
+
```

Comment:
The sentence here should be in close proximity of the sentence https://github.com/ome/ome-documentation/pull/2011/files#diff-3bd9f4e3933d8044e9084726276de4cbR13 - maybe even at a cost of moving the preceding sentence ``All the code....`` with it upwards.